uiautomationutilities/perf_tool/PerfData.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace PerfClTool.Measurement
+{
+    internal class PerfData
+    {
+        public List<PerfDataRecord> PerfDataRecordsList = new List<PerfDataRecord>();
+        private PerfDataRecord _headers;
+        //private static readonly string s_perfMarkersNameMappingFile = Path.Combine("codemarkersMapping.csv");
+        public PerfData(string filePath)
+        {
+            if (!File.Exists(filePath))
+            {
+                throw new Exception($"File not found {filePath}");
+            }
+            PerfDataRecordsList = File.ReadLines(filePath)
+                .Select(line => line.Split(','))
+                .Select(tokens => new PerfDataRecord(
+                    tokens[0], //TimeStamp
+                    tokens[1], //Marker
```

Comment:
what are chances of changing this headers order?
if its too frequent I would say do string comparison and return index from from a utility.
int MapHeaderToIndex(token)
{

> switch(token):
>>case "TimeStamp":
>>>return 0;
}

Reply:
Taken care of this.